### PR TITLE
Improve UX on message feature

### PR DESCRIPTION
**Pull request by anandarh**

### CODE REVIEW

File: lib/screens/messages/messages.dart [hunk 1]
```diff
@@ -227,19 +227,20 @@ class _MessagesState extends State<Messages> {
...
+                                  fontSize: 14.0,
+                                  height: Dimens.lineHeight,
                                   color: ColorBase.grey800,
                                   fontWeight: hasRead
                                       ? FontWeight.w400
-                                      : FontWeight.w900),
+                                      : FontWeight.w700),
                             ),
                             Container(
                               margin: EdgeInsets.only(top: 8.0, bottom: 13.0),
```
Comment (gaganugraha33) [inline lib/screens/messages/messages.dart:238]: add `const` inside margin @anandarh

File: lib/screens/messages/messagesDetailSecreen.dart [hunk 2]
```diff
@@ -189,7 +189,7 @@ class _MessageDetailScreenState extends State<MessageDetailScreen> {
               ),
             ),
             SizedBox(
```
Comment (gaganugraha33) [inline lib/screens/messages/messagesDetailSecreen.dart]: add `const` in SizedBox @anandarh

File: lib/screens/messages/messagesDetailSecreen.dart [hunk 3]
```diff
@@ -198,8 +198,9 @@ class _MessageDetailScreenState extends State<MessageDetailScreen> {
             right: 0.0,
             bottom: 0.0,
             child: data.actionTitle != null && data.actionUrl != null
-                ? Padding(
+                ? Container(
                     padding: EdgeInsets.all(20.0),
```
Comment (gaganugraha33) [inline lib/screens/messages/messagesDetailSecreen.dart]: this padding too @anandarh

**Comment by anandarh** [comment]
Done gan @gaganugraha33

**Review by gaganugraha33 — APPROVED** [review]

**Review by jaluowen — APPROVED** [review]

**Pull request merged**